torch/csrc/jit/register_prim_ops.cpp

```diff
@@ -116,10 +116,10 @@ RegisterOperators reg(
     {Operator(
          "prim::profile(...) -> ()",
          [](const Node* node) {
-           return [node](Stack& stack) {
-             auto addr = node->i(attr::data);
-             std::function<void(Stack&)>& callback =
-                 *reinterpret_cast<std::function<void(Stack&)>*>(addr);
+           // TODO: figure out why cast isn't marked as const
```

Comment:
https://github.com/pytorch/pytorch/blob/master/torch/csrc/jit/ir.h#L587

these TODOs are stacking up

Reply:
Yes. I'm going to create another Github issue for this and assign it to myself. Even if it will be a trivial fix, I'd rather put it in a separate PR.